package.json

```diff
@@ -26,6 +26,7 @@
     "next-redux-wrapper": "^8.1.0",
     "react": "^18.2.0",
     "react-chartjs-2": "^5.2.0",
+    "react-datepicker": "^4.25.0",
     "react-dates": "^21.8.0",
```

Comment:
I hope, we have to also remove the `react-dates` package, so run `yarn remove react-dates` command to remove the package.